iot-hub/Samples/device/X509DeviceCertWithChainSample/readme.md

```diff
@@ -16,15 +16,19 @@ Create a secure string password to use in creation of self-signed certificates.
 
 ### Step 3
 
-Run the [AuthSetup.ps1](https://github.com/Azure-Samples/azure-iot-samples-csharp/blob/master/iot-hub/Samples/device/X509DeviceCertWithChainSample/AuthSetup.ps1) to setup the IoT hub and device with necessary certificates.
+[AuthSetup.ps1](https://github.com/Azure-Samples/azure-iot-samples-csharp/blob/master/iot-hub/Samples/device/X509DeviceCertWithChainSample/AuthSetup.ps1) creates the following certificates, and uploads and verifies the root certificate to your IoT Hub.
+
+Run the following powershell command in admin mode.
```

Comment:
This might be better off as a .ps1 error instead of in the readme :
```
$isAdmin = ([Security.Principal.WindowsPrincipal] [Security.Principal.WindowsIdentity]::GetCurrent()).IsInRole([Security.Principal.WindowsBuiltInRole] "Administrator")
if (-not $isAdmin)
{
    throw "This script must be run in administrative mode."
}
```